### ISSUE TITLE
convert bulk transcriptomes into spatially resolved single-cell expression profile

### ISSUE THREAD

**Issue by Echoloria** [ZJUFanLab/bulk2space#7]
Hi,
  I'm new to bulk2space, and I only have bulk RNAseq data from mouse brain which was made by our lab. I want to convert bulk transcriptomes into spatially resolved single-cell expression profile. I know how to convert bulk RNAseq data into single cell data. Here are my questions:
1. How to get the spatial information from my bulk RNAseq data, do I have to do some experiments about spatial information by Laser capture microdissection (LCM) technology?
2. since my bulk RNAseq data are form brain tissue, tissues contain many layers of cells. How do you distinguish between different layers of cells? Or do I have to do bulk RNAseq from single layers?

Thanks,
Echo.

**Comment by SpaTrek** [comment]
Hi, Thx for using our Bulk2Space tool.
1. Bulk2Space needs two references, single-cell and spatial reference, to conduct spatial deconvolution. Specifically, since you have already got bulk RNA-seq data of the mouse brain, I suggest you search for the single-cell and spatial transcriptomics profiles from GEO or other resources. You don't really need to do experiments on single-cell or spatial, just use public datasets.
2. If you do want to use self-obtained spatial information, I strongly recommend NOT using LCM because of the lower sample throughput. In our manuscript, the data derived from LCM are used as bulk transcriptome, not spatial reference.
3. It is cool when your bulk RNA-seq data are heterogeneous, you don't need to sequence each layer of the brain tissue, the Bulk2Space is designed for finding the spatial variation in complex tissues, the layered structure is definitely in this scope. Maybe the fourth result of our manuscript can answer your question. We used LCM to acquire the whole isocortex  tissue, containing six layers and obtained RNA-seq data from the heterogeneous tissue. Then, we downloaded the single-cell and spatial reference from publicly available resources such as 10X Genomics database. After spatial deconvolution of the bulk RNA-seq, we reconstructed the layered structure of the isocortex.
Just make full use of the existed resources. I believe that the spatial transcriptomics data of the mouse brain are everywhere.
